qiskit/circuit/quantumcircuit.py

```diff
@@ -6875,7 +6875,10 @@ def qubit_stop_time(self, *qubits: Union[Qubit, int]) -> float:
             if len(qubits) == len([done for done in dones.values() if done]):  # all done
                 return max(stop for stop in stops.values())
 
-        return 0  # If there are no instructions over bits
+        if len(stops) > 0:  # not all but some qubits has instructions
+            return max(stop for stop in stops.values())
```

Comment:
I think this list-comprehension is unnecessary:
```suggestion
            return max(stops.values())
```